.cookiecutter/cookiecutter.json

```diff
@@ -29,6 +30,7 @@
     "postgres": "yes",
     "docker": "yes",
     "__postgres_port": "5433",
+    "__postgres_version": "11.5-alpine",
```

Comment:
`__postgres_version` was missing from LMS's `cookiecutter.json` so it was just getting the cookiecutter's default version which happened to be `11.5-alpine`. The cookiecutter's default version has now been upgraded, so adding this here to pin LMS's version as we don't want to upgrade it yet